src/mixpanel-core.js

```diff
@@ -758,7 +758,7 @@ MixpanelLib.prototype._track_or_batch = function(options, callback) {
  *
  * @param {String} event_name The name of the event. This can be anything the user does - 'Button Click', 'Sign Up', 'Item Purchased', etc.
  * @param {Object} [properties] A set of properties to include with the event you're sending. These describe the user who did the event or details about the event itself.
- * @param {Object} [options] Optional configuration for this track request.
+ * @param {Object} [options] Optional configuration for this track request; valid options are 'transport' and/or 'send_immediately'
```

Comment:
Not sure this is a useful addition, given that these two options are listed just below. Maybe something more general like "available options are listed below." FWIW this is what it currently looks like in the docs:

<img width="752" alt="Screen Shot 2022-10-11 at 11 37 26 AM" src="https://user-images.githubusercontent.com/855881/195172578-33c46bfe-e515-4580-9a95-d524c328df9a.png">

Reply:
indeed. saw that. guess naming stuff is hard. will withdraw this.